openpgp-dsm/src/lib.rs

```diff
@@ -616,20 +616,32 @@ impl KeyMetadata {
     }
 }
 
+pub enum KeyFlagsArg {
```

Comment:
I'm not sure if this is worthwhile, but could we instead make `KeyFlagsArg` a bitflag instead of an enum that encompasses various combinations? E.g., what if I want to encode some novel subkey flag combo?

Reply:
anyway this probably isn't super important

Reply:
Any reason we are not using the existing `KeyFlags` struct? E.g. `Vec<KeyFlags>` instead of this enum, and parsing happens earlier.

Reply:
Switched to `KeyFlags` and removed the newly added `KeyFlagArgs`.

Reply:
imma resolve this, since it seems like you've made the fix as suggested by zugzwang